File: L004/Patterns/pattern1.java
import java.util.Scanner;

public class pattern1{

    public static Scanner sc = new Scanner(System.in);

    public static void pattern1_(int row){
        int nst=1;
        for(int r=1;r<=row;r++){
            for(int cst=1;cst<=nst;cst++){
                System.out.println("*\t");
            }
            nst++;
            System.out.println();
        }
    }
    
    public static void main(String []args){
    
        int n = sc.nextInt();
    
        pattern1_(n);
        
    }
}

File: L004/Patterns/pattern1.cpp
#include<iostream>
using namespace std;


void pattern1_(int row){
    int nst=1;
    for(int r=1;r<=row;r++){
        for(int cst=1;cst<=nst;cst++){
            cout<<"*\t";
        }
        nst++;
        cout<<endl;
    }
}

int main(){

    int n;
    cin>>n;

    pattern1_(n);
    return 0;
}